odoo/modules/registry.py

```diff
@@ -892,7 +892,7 @@ def signal_changes(self):
             _logger.info("Registry changed, signaling through the database")
             with closing(self.cursor()) as cr:
                 cr.execute("select nextval('base_registry_signaling')")
-                self.registry_sequence = cr.fetchone()[0]
+                self.registry_sequence += 1
```

Comment:
I think this line and the one below deserve a comment like
```suggestion
                # in case another process concurrently increments the sequence in
                # database, the next call to check_signaling() will detect that
                # the registry sequence is actually out-of-date
                self.registry_sequence += 1
```
You may either duplicate and adapt, or factor out this comment to explain why all local sequences are simply incremented.

It may also be worth noting (in the description) that, if there is no concurrent sequence update, this behaves exactly like before, and will therefore not cause any extra registry/cache invalidation.